src/main/java/com/chatapp/vartalaab/redisEntity/UserSessionDetails.java

```diff
@@ -0,0 +1,27 @@
+package com.chatapp.vartalaab.redisEntity;
+
+import java.util.Map;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.redis.core.RedisHash;
+
+import com.chatapp.vartalaab.utils.GeneralUtility;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@RedisHash("user_session")
+@Data
+@AllArgsConstructor
+public class UserSessionDetails {
+    
+    @Id
+    private String userSessionKey;
+    private int activeSessions;
```

Comment:
activeSessions var is not necessary, since we have a map of tokens to get activeCount, but intuitively it looked good, so I kept it. Any comments on this ?